### PR TITLE
Disable stderr output in Release builds

### PR DESCRIPTION
**Pull request by asomsiko**

### CODE REVIEW

File: src/frontends/paddle/src/frontend.cpp [hunk 1]
```diff
@@ -392,5 +393,9 @@ PADDLE_C_API void* GetFrontEndData() {
     res->m_creator = []() {
         return std::make_shared<ov::frontend::paddle::FrontEnd>();
     };
+#ifndef OPENVINO_DEBUG_ENABLE
+    // disable protobuf logging
+    google::protobuf::SetLogHandler(nullptr);
```
Comment (asomsiko) [inline src/frontends/paddle/src/frontend.cpp:398]: google::protobuf::SetLogHandler is not thread-safe. I expect the GetFrontEndData call does not cause race conditions.

**Review by tomdol — APPROVED** [review]
I don't recall any logs being reported from protobuf when importing ONNX models but I suppose it's fine to disable them explicitly. Is there any particular reason we should see even error logs from OV in release mode?

**Comment by tomdol** [comment]
In particular the user will not get any error messages in case unregistering operators in ONNX FE fails.

**Review by ilyachur — APPROVED** [review]

**Comment by ilya-lavrenov** [comment]
@ilyachur is it still valid?

**Comment by ilyachur** [comment]
Yes it is valid I will port these changes to the latest master

**Comment by ilyachur** [comment]
Created new PR with the same changes

**Pull request closed without merging**